test/Edges.java

```diff
@@ -0,0 +1,52 @@
+import static org.junit.Assert.*;
+import org.junit.*;
+import static org.mockito.Mockito.*;
+import java.util.*;
+import java.io.*;
+
+public class Edges
```

Comment:
These tests won't work if there are further changes, since they are just comparing already-output files.  Tests should be checking the .toString() output directly - what happens if I make further changes to .toString(), I can't use these tests anymore!